package.json

```diff
@@ -358,6 +358,7 @@
     "fetch-mock": "7.3.0",
     "geckodriver": "^1.16.1",
     "getopts": "2.0.0",
+    "github-xyz": "^0.0.21",
```

Comment:
this needs to be updated to @elastic/github-checks-reporter once its properly published